src/coll/batch.rs

```diff
@@ -153,16 +175,28 @@ impl Batch {
             }
             Batch::Update(ref mut models) => {
                 match model {
-                    WriteModel::ReplaceOne { filter, replacement: update, upsert } |
-                    WriteModel::UpdateOne { filter, update, upsert } => {
+                    WriteModel::ReplaceOne {
```

Comment:
Okay, this is weird and unnecessary. I guess rustfmt isn't as improved as I hoped.